std/path.d

```diff
@@ -1073,19 +1073,11 @@ unittest
     ---
 */
 immutable(Unqual!C1)[] defaultExtension(C1, C2)(in C1[] path, in C2[] ext)
-    @trusted pure // TODO: nothrow (because of to())
+    @trusted
```

Comment:
This shouldn't be trusted seeing as it's generic, but I guess it's no worse than before. What happened to pure?